client/js/place.js

```diff
@@ -456,6 +457,34 @@ var place = {
         this.didSetHash = true;
     },
 
+    contextMenu: function(event) {
+        event.preventDefault();
+        let zoom = place._getZoomMultiplier();
+        let x = Math.round((event.pageX - $(place.cameraController).offset().left) / zoom)
+        let y = Math.round((event.pageY - $(place.cameraController).offset().top) / zoom)
```

Comment:
Look at how we capture all the other events using another function, it makes it so that we can get the internal object's variables (without breaking OOP), and still get the event passed to us